ucb_prefect_tools/util.py

```diff
@@ -146,16 +148,63 @@ def deployable(flow_obj):
+    flow, run it via Prefect Cloud, then delete the deployment afterward; as such, it is useful
+    for testing.
+
+    Your git repo must be "clean" in order to deploy, meaning you have no uncommitted changes and
+    you are up to date with your remote branch. This is because Prefect will pull your code from
+    Github when running the flow.
+
+    Command-line options:
+
+        `--image-name`: Change what image to use for deployment from what is specified in the flow
+            docstring
+        `--image-branch`: Change the image branch, aka mage label, to use. By default, when you are
+            on the `main` git branch, this will be `main`, and when you are on any other branch,
+            this will be the same as the branch name. See the oit-ds-tools-prefect-images repo for
+            information about building images.
+        `--label`: This determines what work pool will be used for the deployment. Options are
+            `main` or `dev`, and the default is based on the current git branch. If you deploy a
+            main flow with the dev label, it will act like a dev deployment with no schedule, error
+            notifications, etc. If you deploy a dev flow with a main label, it will run just like
+            a dev deployment except using the main namespace in Kubernetes. You could also create
+            a new work pool/namespace and point the deployment at it using this option.
+
+    When deploying a flow, the following Sphinx-style docstring fields are used to define certain
+    parameters (see oit-ds-flows-template for an example):
+
+        main_params: A json-style dict of parameter overrides to use for flow runs when deployed
+            from the main branch. At the very least, the env param should be overridden to "prod"
+            from it's usual default of "dev".
+        schedule: A cron string giving the schedule to be applied to the main-branch deployment.
+            This will use the common America/Denver timezone used by ucb_prefect_tools.
+        image_name: The image to use for this flow, usually just oit-ds-prefect-default.
+        tags: (Optional) Any additional tags to apply to main-branch deployments of this flow. At
+            least list the autotest tag for flows which can safely be run with default (i.e. dev)
+            parameters as part of automated testing.
+        source_systems: An informal list of systems from which this flow pulls data. Make it helpful
+            to non-engineers: e.g. instead of "CUTransfer", say "Bookstore (CUTransfer)"
+        sink_systems: Same as source systems, but for places where the data is being sent.
+        customer_contact: A comma-separated list of email addresses of people or teams to reach out
+            to if we have questions about the flow (typically on the "sink" side). Feel free to
+            include names or other info when needed for clarity.
+
+    """
```

Comment:
```suggestion

        source_systems: An informal list of systems from which this flow pulls data. Make it helpful
            to non-engineers: e.g. instead of "CUTransfer", say "Bookstore (CUTransfer)"

        sink_systems: Same as source systems, but for places where the data is being sent.

        customer_contact: A comma-separated list of email addresses of people or teams to reach out
            to if we have questions about the flow (typically on the "sink" side). Feel free to
            include names or other info when needed for clarity.

    """
```

> Feel free to include names or other info when needed for clarity.

Might be overkill, but should we advise folks to specify them as comma-separated tuples? E.g., "(mathplacement@colorado.edu, Silva Chang)", when the email doesn't indicate who the actual person is? Or another format? JSON?

Reply:
I think the customer_contact will just be a human-readable field for now. I don't think we need to worry about being able to parse it.